.gitignore

```diff
@@ -168,9 +167,9 @@ src/ledgermonolith/istio/istio-1.*
 register-key.json
 
 # ASM
-install_asm*
-extras/asm-multicluster/asm-cluster-1/*
-extras/asm-multicluster/asm-cluster-2/*
+asmcli*
```

Comment:
Do we have to include this as part of prereq tools, or is that included somewhere?

Reply:
It is downloaded as part of the [cluster-setup.sh](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/771/files#diff-d4d16f85ecfbac8a86625068d9df5bad897713b1899d5674aa09e7bcf9e2b863) line 109

Reply:
Gotcha.